I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/helpers.rpy", line 142, in script
    python:
ScriptError: could not find label ''.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/home/moemen/gradproj/RenPy/renpy-7.4.11-sdk/renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "/home/moemen/gradproj/RenPy/renpy-7.4.11-sdk/renpy/main.py", line 662, in main
    run(restart)
  File "/home/moemen/gradproj/RenPy/renpy-7.4.11-sdk/renpy/main.py", line 148, in run
    renpy.execution.run_context(True)
  File "/home/moemen/gradproj/RenPy/renpy-7.4.11-sdk/renpy/execution.py", line 922, in run_context
    context.run()
  File "game/helpers.rpy", line 142, in script
    python:
  File "/home/moemen/gradproj/RenPy/renpy-7.4.11-sdk/renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label ''.

Linux-5.4.0-105-generic-x86_64-with-debian-bullseye-sid
Ren'Py 7.4.11.2266
RenPyTest 1.0
Fri Apr  8 17:34:27 2022
